java/pom.xml

```diff
@@ -72,7 +75,8 @@
         <!-- The version of Arrow should match the version used by Spark, under spark-sql-api. -->
         <arrow.version>12.0.1</arrow.version>
         <!--
-        Spark Catalyst uses an old version of DataSketches. This must match the version declared there, as it is
+        Spark Catalyst uses an old version of DataSketches. This must match the version declared there, as
+        it is
         provided in EMR.
```

Comment:
This seems to have reformatted poorly. There are a lot of cases like this in this file.